### PR TITLE
GG-27017 Fixed availability of "Add user" action.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js [hunk 1]
```diff
@@ -20,9 +20,39 @@ import * as admin from '../../page-models/pageAdminListOfRegisteredUsers';
...
+const _checkListOfActions = async(t, actions, excluded) => {
+    const actionSelector = Selector('.dropdown-menu a');
+
+    for (let i = 0; i < actions.length; i++) {
+        const actionLbl = actions[i];
+
+        if (excluded)
+            await t.expect(actionSelector.withText(actionLbl).exists).notOk();
+        else
+            await t.expect(actionSelector.withText(actionLbl).exists).ok();
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]:
```js
await t.expect(actionSelector.withText(actionLbl).exists == excluded).notOk();
```

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js [hunk 2]
```diff
@@ -61,3 +91,82 @@ test('Become user', async(t) => {
...
+
+    await _changeUserSelection('User Name');
+
+    await admin.usersTable.performAction('Revoke admin');
+
+    await t.expect(successNotification.withText('Admin rights was successfully revoked for user: "User Name"').exists).ok();
+
+    await admin.usersTable.performAction('Grant admin');
+
+    await t.expect(successNotification.withText('Admin rights was successfully granted for user: "User Name"').exists).ok();
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: There's no validation that admin permissions were granted/revoked - alert does not guarantee that

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js [hunk 3]
```diff
@@ -61,3 +91,82 @@ test('Become user', async(t) => {
...
+});
+
+test("Remove user", async(t) => {
+    await t.navigateTo(resolveUrl('/settings/admin'))
+        .click(admin.userNameCell.withText('User Name'));
+
+    await admin.usersTable.performAction('Remove user');
+
+    await t.expect(confirmation.body.exists).ok();
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: We assert that modal is displayed but we don't assert the content of the modal

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js (hunk 1 above)
Comment (Klaster1) [reply modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: TestCafe-specific promises don't work like that.

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js (hunk 3 above)
Comment (Klaster1) [reply modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: Agree with @MaxLosevskoy, I'd at least asset the confirmation text.

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js (hunk 1 above)
Comment (vsisko) [reply modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: Changed check of action list.

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js (hunk 2 above)
Comment (vsisko) [reply modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: Added check of admin status change.

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js (hunk 3 above)
Comment (vsisko) [reply modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: Added check of confirm dialog message.

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js [hunk 4]
```diff
@@ -61,3 +93,89 @@ test('Become user', async(t) => {
...
+    await t.eval(() => window.location.reload());
+    await t.expect(admin.userNameCell.withText(OTHER_USER).find('i.icon-user').exists).ok(LONG_TIMEOUT_OPTION);
+
+    await _changeUserSelection(OTHER_USER);
+    await admin.usersTable.performAction('Grant admin');
+
+    await t.expect(successNotification.withText(`Admin rights was successfully granted for user: "${OTHER_USER}"`).exists).ok();
+
+    await t.eval(() => window.location.reload());
+    await t.expect(admin.userNameCell.withText(OTHER_USER).find('i.icon-admin').exists).ok(LONG_TIMEOUT_OPTION);
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: Why lines 141 and 149 are equal? In 1st case user shouldn't be an admin and vice versa in 2nd case

File: modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js (hunk 3 above)
Comment (Klaster1) [reply modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js]: Looks good to me.

**Pull request merged**